exercises/resistor-significants/metadata.yml

```
@@ -0,0 +1,4 @@
+---
+blurb: "Convert the colors of a resistor's first two bands to the significant figures of its value in Ohms."
```

```suggestion
blurb: "Convert color codes, as used on resistors, to a numeric value."
```
Or leave the resistor out here?

---

I think it's fine to leave it in here.